langchain4j-core/src/main/java/dev/langchain4j/model/image/ImageModel.java

```diff
@@ -29,4 +29,30 @@ public interface ImageModel {
     default Response<List<Image>> generate(String prompt, int n) {
         throw new IllegalArgumentException("Operation is not supported");
     }
+
+    /**
+     * Given an existing image, edit this image following the given prompt.
+     *
+     * @param prompt The prompt to edit the image.
+     * @param image The image to be edited.
+     * @return The generated image Response.
+     */
+    default Response<Image> edit(String prompt, Image image) {
```

Comment:
Funny! At first, I had the prompt at the end, and then thought that it would look more like the generate methods if prompt were first. But I agree, it sounds more logical to move it at the end!